.gitignore

```diff
@@ -0,0 +1 @@
+.obsidian
```

Comment:
Needs a newline at end of file.

Also, whoopsies for me leaving the folder in. I guess anyone who downloads or clones now has to deal with manually replacing this config with their own.